examples/snowpack/src/pages/tutorials/react.md

```diff
@@ -118,7 +118,7 @@ The `mount` configuration changes where Snowpack looks for and builds files. Eve
 ```diff
   mount: {
 -   /* ... */
-+   // directory name: 'build directory',
```

Comment:
I think the comma makes sense, I believe it's expected that removing the `//` will produce working code.